openedx_lti_tool_plugin/deep_linking/tests/test_views.py

```diff
@@ -0,0 +1,277 @@
+"""Tests views module."""
+from unittest.mock import MagicMock, PropertyMock, patch
+from uuid import uuid4
+
+from django.test import RequestFactory, TestCase
+from django.urls import reverse
+from pylti1p3.exception import LtiException
+
+from openedx_lti_tool_plugin.apps import OpenEdxLtiToolPluginConfig as app_config
+from openedx_lti_tool_plugin.deep_linking.exceptions import DeepLinkingException
+from openedx_lti_tool_plugin.deep_linking.tests import MODULE_PATH
+from openedx_lti_tool_plugin.deep_linking.views import DeepLinkingFormView, DeepLinkingView
+
+MODULE_PATH = f'{MODULE_PATH}.views'
+
+
+@patch.object(DeepLinkingView, 'tool_config', new_callable=PropertyMock)
+@patch.object(DeepLinkingView, 'tool_storage', new_callable=PropertyMock)
+@patch(f'{MODULE_PATH}.DjangoMessageLaunch')
+@patch(f'{MODULE_PATH}.validate_message')
+@patch(f'{MODULE_PATH}.redirect')
+class TestDeepLinkingViewPost(TestCase):
+    """Test ResourceLinkLaunchView post method."""
+
+    def setUp(self):
+        """Set up test fixtures."""
+        super().setUp()
+        self.view_class = DeepLinkingView
+        self.factory = RequestFactory()
+        self.url = reverse('1.3:deep-linking:root')
+        self.request = self.factory.post(self.url)
+
+    def test_with_deep_linking_request(
```

Comment:
Could you please rename this case so it is clear that it's the happy path? Or specify it in the docstring.

Reply:
@alexjmpb How do you suggest it should be renamed or how could be specified on the docstring?

Reply:
I would say it can be renamed to test_post or putting in the docstring a description that clarifies that this case tests that the post method is being performed correctly.